lua/gdb/backend/lldb.moon

```diff
@@ -39,6 +39,7 @@ backend =
     initScm: LldbScm
     delete_breakpoints: 'breakpoint delete'
     breakpoint: 'b'
+    print: 'frame var'
```

Comment:
This key might be ignored, because the one, which is used for evaluation, is `print %s`. That's how pdb is able to execute `print(x)`. Could you please fix that and try whether the tests still pass?